### PR TITLE
Component feature

### PR DESCRIPTION
**Pull request by romainbazeler**
add button
add footer

### CODE REVIEW

**Review by lcskoerner — CHANGES_REQUESTED** [review]
.env is already in .gitignore no need to re-add

File: app/assets/stylesheets/components/_footer.scss [hunk 1]
```diff
@@ -0,0 +1,9 @@
+.footer {
+  // background-color: $white-button;
+  position: absolute;
+  width: 44px;
+  height: 44px;
+  left: 41px;
+  top: 743px;
+  border-radius: 50px;
+}
```
Comment (lcskoerner) [inline app/assets/stylesheets/components/_footer.scss:9]: pay attention to the end of files. Github will ask you to add one extra line.

File: app/assets/stylesheets/config/_colors.scss [hunk 2]
```diff
@@ -0,0 +1,2 @@
+$white-button: #F3F8FA;
+$grey-button: #333333;
```
Comment (lcskoerner) [inline app/assets/stylesheets/config/_colors.scss:2]: please add extra line

File: app/assets/stylesheets/config/_index.scss [hunk 3]
```diff
@@ -0,0 +1,2 @@
+@import "fonts";
+@import "colors";
```
Comment (lcskoerner) [inline app/assets/stylesheets/config/_index.scss:2]: please add extra line

File: app/assets/stylesheets/components/_footer.scss (hunk 1 above)
Comment (romainbazeler) [reply app/assets/stylesheets/components/_footer.scss:9]: Oh ok I see. I will add an extra line today. Thanks!

**Pull request merged**